### PR TITLE
Xmlreader admin notice

### PR DESCRIPTION
**Pull request by maheshwaghmare**

### CODE REVIEW

File: inc/classes/class-astra-sites-page.php [hunk 1]
```diff
@@ -93,6 +93,23 @@ static public function init_admin_settings() {
...
+		 * Admin notice
+		 *
+		 * @since 1.2.8
+		 */
+		public static function notices() {
+			if ( ! class_exists( 'XMLReader' ) ) {
+				?>
+				<div class="notice astra-sites-xml-notice notice-error">
+					<p><b><?php _e( 'Required XMLReader PHP extension is missing on your server!', 'astra-sites' ); ?></b></p>
+					<p><?php _e( 'Astra Sites import requires XMLReader extension to be installed. Please contact your web hosting provider and ask them to install and activate the XMLReader PHP extension.', 'astra-sites' ); ?></p>
```
Comment (Nikschavan) [inline inc/classes/class-astra-sites-page.php]: Use white-labeled string in place of `Astra`

File: inc/classes/class-astra-sites-importer.php [hunk 2]
```diff
@@ -146,6 +146,10 @@ function prepare_xml_data() {
 
 			do_action( 'astra_sites_import_prepare_xml_data' );
 
+			if ( ! class_exists( 'XMLReader' ) ) {
+				wp_send_json_error( __( 'The XMLReader is not enabled on your server. To enable it please contact your server administrator.', 'astra-sites' ) );
```
Comment (Nikschavan) [inline inc/classes/class-astra-sites-importer.php]: If XMLReader is not available, it imports all other settings and only skips XML import. This creates an incomplete website. We should bail early and not import anything if this is not present.

Comment (maheshwaghmare) [reply inc/classes/class-astra-sites-importer.php]: Updated the message.

File: inc/classes/class-astra-sites-page.php (hunk 1 above)
Comment (maheshwaghmare) [reply inc/classes/class-astra-sites-page.php]: Used white label name.

**Pull request merged**